cul.js

```diff
@@ -27,7 +27,24 @@ try {
 }
 
 adapter.on('stateChange', function (id, state) {
-    //if (cul) cul.cmd();
+    if(state.from != "system.adapter." + adapter.namespace) {
+        adapter.log.debug("my namespace: " + adapter.namespace);
+        adapter.log.debug("State Change " + JSON.stringify(id) + " State: " + JSON.stringify(state));
+        //  State Change "cul.0.FS20.123401.cmd" State: {"val":2,"ack":false,"ts":1581365531968,"q":0,"from":"system.adapter.admin.0","user":"system.user.admin","lc":1581365531968}
+        var oAddr = id.split('.');
+        // 0: cul; 1:0; 2:FS20; 3:123401; 4:cmd;
+        var sHousecode = oAddr[3].substring(0,4);
+        var sAddress = oAddr[3].substring(4,6);
+        switch (oAddr[4]) {
+            case "cmdRaw":
+                sendCommand({protocol: oAddr[2], housecode: sHousecode, address: sAddress, command: state.val});
```

Comment:
is only FS20 supported? If yes it should be checked that it is only used for this.

Reply:
Completed with commit https://github.com/ioBroker/ioBroker.cul/pull/11/commits/c980c3373d12a5cc568c23e4479d1b5b0d8da877